process/block/shardblock_request_test.go

```diff
@@ -0,0 +1,586 @@
+package block_test
```

Comment:
rename the file to shardblockRequest_test ?

Reply:
done